### ISSUE TITLE
[Bug] v.import gets stuck on old-style .gfs file during GML import

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass#382]
Due to the fact that GDAL 2.x appears to write a temporary .gfs file during GML import, it is not possible to import e.g. the cloud mask file shipped with a Sentinel-2 scene.

(Note: the problem no longer occurs with GDAL 3.x, tested with GDAL 3.0.4)

GML file to test:
* [MSK_CLOUDS_B00_gml.zip](https://github.com/OSGeo/grass/files/4258709/MSK_CLOUDS_B00_gml.zip)

.gfs file (auto-generated with GDAL 3.0.4):
* [MSK_CLOUDS_B00_gfs_gdal304.zip](https://github.com/OSGeo/grass/files/4258713/MSK_CLOUDS_B00_gfs_gdal304.zip)

```
# create UTM location
grass78 -c epsg:32749 ~/grassdata/utm_32749

# attempt to import cloud mask GML file
GRASS 7.8.3dev (utm_32749):~ > v.import MSK_CLOUDS_B00.gml 
ERROR: Unable to convert input map projection to GRASS format; cannot
       create new location.
ERROR: Unable to create location from OGR datasource <MSK_CLOUDS_B00.gml>

# now tricking it : -)
GRASS 7.8.3dev (utm_32749):~ > cp MSK_CLOUDS_B00.gfs_gdal304 MSK_CLOUDS_B00.gfs

# and import works fine with a proper .gfs file:
GRASS 7.8.3dev (utm_32749):~ > v.import MSK_CLOUDS_B00.gml 
Check if OGR layer <MaskFeature> contains polygons...
 100%
Creating attribute table for layer <MaskFeature>...
Importing 920 features (OGR layer <MaskFeature>)...
[...]
Attaching centroids...
 100%
Input <MSK_CLOUDS_B00.gml> successfully imported without reprojection
```

**Now, what is the difference in the .gfs file bewteen GDAL 2.3.2 and GDAL 3.0.4?**

```
diff -u MSK_CLOUDS_B00.gfs MSK_CLOUDS_B00.gfs_gdal304
--- MSK_CLOUDS_B00.gfs	2020-02-27 00:33:03.445970880 +0100
+++ MSK_CLOUDS_B00.gfs_gdal304	2020-02-26 09:49:10.623901959 +0100
@@ -4,6 +4,7 @@
     <ElementPath>MaskFeature</ElementPath>
     <!--POLYGON-->
     <GeometryType>3</GeometryType>
+    <SRSName>urn:ogc:def:crs:EPSG:8.7:32749</SRSName>
     <DatasetSpecificInfo>
       <FeatureCount>920</FeatureCount>
       <ExtentXMin>699960.00000</ExtentXMin>
```

... the SRSName tag has been introduced which makes things work.

Proof:
```
gdalinfo --version
GDAL 2.3.2, released 2018/09/21

# 1st run (successful due to absence of old-style .gfs file):
rm -f MSK_CLOUDS_B00.gfs
ogrinfo -so -al MSK_CLOUDS_B00.gml
INFO: Open of `MSK_CLOUDS_B00.gml'
      using driver `GML' successful.
Metadata:
  NAME=MSK_CLOUDS pixels mask from data-strip S2A_OPER_MSK_CLOUDS_SGS__20200223T061113_A024396_T49MGU_B00_MSIL1C
Layer name: MaskFeature
Geometry: Polygon
Feature Count: 920
Extent: (699960.000000, 9790240.000000) - (809760.000000, 9900040.000000)
Layer SRS WKT:
PROJCS["WGS 84 / UTM zone 49S",
    GEOGCS["WGS 84",
        DATUM["WGS_1984",
[...]
    AUTHORITY["EPSG","32749"]]
gml_id: String (0.0) NOT NULL
maskType: String (6.0)

# 2nd run (failing, since the first run created the .gfs file lacking the SRSName tag:
ogrinfo -so -al MSK_CLOUDS_B00.gml
INFO: Open of `MSK_CLOUDS_B00.gml'
      using driver `GML' successful.
Metadata:
  NAME=MSK_CLOUDS pixels mask from data-strip S2A_OPER_MSK_CLOUDS_SGS__20200223T061113_A024396_T49MGU_B00_MSIL1C

Layer name: MaskFeature
Geometry: Polygon
Feature Count: 920
Extent: (699960.000000, 9790240.000000) - (809760.000000, 9900040.000000)
Layer SRS WKT:
(unknown)
gml_id: String (0.0) NOT NULL
maskType: String (6.0)
```

**Expected behavior**
v.import should not fail during a GML import when having GDAL 2.x on the machine.

**Discussion**
Indeed the problem is more on the GDAL side than on the GRASS GIS side but it would take a long time to see an updated GDAL being widely available. Hence, a short term solution would be to somehow ignore the .gfs file during import (since not relevant anyway).

I tried to find in the GDAL GML driver where the .gfs is generated to see through `git blame` when the needed SRSNAME tag was introduced but I didn't figure it out... anyone? (maybe @rouault ?)

**Comment by rouault** [comment]
SRSName presumably written since https://github.com/OSGeo/gdal/commit/a43362ee372a657ec2714e9beb8945271c359b67 that went in 2.4.1 and 3.0.0

**Comment by neteler** [comment]
So, an approach might be to modify `v.import` and 

- if format is GML && GDAL < 2.4.1
   - then inject the SRSName tag in the .gfs file, probably here:

https://github.com/OSGeo/grass/blob/8eba412c83d8f1af8bc4e506e9128034ebc68dc3/scripts/v.import/v.import.py#L256

**Comment by AnikaBettge** [comment]
It has to be changed addational in https://github.com/OSGeo/grass/blob/8eba412c83d8f1af8bc4e506e9128034ebc68dc3/scripts/v.import/v.import.py#L211

**Comment by neteler** [comment]
fixed by PR #386
